### DISCUSSION TITLE
Add CodeIgniter

### DISCUSSION THREAD

**Discussion by iyinusa** [Codeception/Codeception#6607] [Ideas]
I believe adding CodeIgniter will be great

**Comment by Naktibalda** [comment]
You are welcome to implement it.
It shouldn't be difficult to create module for Codeigniter 4, connector must create instance of CodeIgniter class, populate and pass Request object and transform response to Symfony BrowserKin Response, [see example](https://github.com/Codeception/module-laminas/blob/ad1fa0ee34e5bf7f1650d293e28e4136fa0b2996/src/Codeception/Lib/Connector/Laminas.php#L55-L112)

**Comment by DavertMik** [comment]
It's 2022, I had requests to get CodeIgniter on board since 2012!
And no one actually did anything about it
CodeIgniter stopped being a significunt PHP web framework a decade ago. So I don't think we will ever add it.
But again, pull requests are open.
